test/psych/test_merge_keys.rb

```diff
@@ -117,7 +117,7 @@ def test_missing_merge_key
 bar:
   << : *foo
       eoyml
-      exp = assert_raise(Psych::BadAlias) { Psych.load yaml }
+      exp = assert_raise(Psych::BadAlias) { Psych.load(yaml, aliases: true) }
```

Comment:
Turns out this test never actually failing because of the `foo` anchor not existing.

Aliases just weren't enabled xD